graphql_compiler/compiler/emit_sql.py

```diff
@@ -276,10 +408,11 @@ def end_fold(self, alias_generator, from_clause, outer_from_table):
         # for now we only handle folds containing one traversal (i.e. join)
         if len(self.join_info) > 1:
```

Comment:
Why does this line use `self.join_info` whereas just a few lines above we use `self._join_info`?